src/tts.py

```diff
@@ -265,16 +273,34 @@ def check_if_xvasynth_is_running(self):
     def run_xvasynth_server(self):
         try:
             # start the process without waiting for a response
-            subprocess.Popen(f'{self.xvasynth_path}/resources/app/cpython_{self.process_device}/server.exe', cwd=self.xvasynth_path)
-
+            if (self.tts_print == 1):
+                # print subprocess output
+                Popen(f'{self.xvasynth_path}/resources/app/cpython_{self.process_device}/server.exe', cwd=self.xvasynth_path, stdout=None, stderr=None)
+            else:
+                # ignore output
+                Popen(f'{self.xvasynth_path}/resources/app/cpython_{self.process_device}/server.exe', cwd=self.xvasynth_path, stdout=DEVNULL, stderr=DEVNULL)
+            # with sp.stdout:
+            #     self.log_subprocess_output(sp.stdout)
+            # print('precomm')
+            # output, error = sp.communicate()
+            # print('postcomm')
+            # log_subprocess_output(StringIO(output))
+            # exitcode = process.wait() # 0 means success
+                # log_subprocess_output(process.stdout)
+                # exitcode = process.wait() # 0 means success
+                # logger = logging.getLogger('serverLog')
+                # logger.setLevel(logging.WARNING)
         except:
             logging.error(f'Could not run xVASynth. Ensure that the path "{self.xvasynth_path}" is correct.')
-            input('\nPress any key to stop Mantella...')
-            sys.exit(0)
-    
+            raise TTSServiceFailure()
+
+    def log_subprocess_output(self, pipe):
+        for line in iter(pipe.readline, b''): # b'\n'-separated lines
+            logging.log(self.loglevel, '%r', line)
```

Comment:
Unused. There was the possibility to have TTS subprocess logs added to Mantella logs. But since the subprocess is a webserver, it never ends. So this could only be doable with some async code.